src/type/float.ts

```diff
@@ -6,10 +6,14 @@ import common = require('../common');
 import {Type} from '../type';
 
 var YAML_FLOAT_PATTERN = new RegExp(
```

Comment:
synced with the upstream https://github.com/nodeca/js-yaml/blob/master/lib/type/float.js#L6